optuna/integration/tensorflow.py

```diff
@@ -19,11 +19,9 @@
 class TensorFlowPruningHook(SessionRunHook):
     """TensorFlow SessionRunHook to prune unpromising trials.
 
-    Example:
-
-        See `the example <https://github.com/optuna/optuna/blob/30dba8814d332ff4
-        0d9c7465ee660a8ecf499547/examples/tensorflow_estimator_simple.py>`_
-        if you want to add a pruning SessionRunHook for TensorFlow's Estimator.
+    See `the example <https://github.com/optuna/optuna/blob/master/examples/
+    pruning/tensorflow_estimator_integration.py>`_
+    if you want to add a pruning SessionRunHook for TensorFlow's Estimator.
```

Comment:
```suggestion
    if you want to add a pruning hook to TensorFlow's estimator.
```
C.f. https://github.com/optuna/optuna/pull/935